src/reducers/dashboards.js

```diff
@@ -117,31 +125,32 @@ export const sGetById = (state, id) =>
  * @param {Object} state The current state
  * @returns {Array}
  */
-export const sGetItems = state => sGetFromState(state).items;
+export const sGetDashboardItems = state => sGetFromState(state).items;
 
 // selector level 2
 
 /**
- * Generic selector which returns filtered dashboards
+ * Generic selector which returns dashboards filtered by a prop value
+ * Uses === to check equality
  *
  * @function
  * @param {Object} state The current state
- * @param {Object} propName The name of the filter prop
- * @param {Object} value The value of the filter prop
+ * @param {String} propName The name of the filter prop
+ * @param {Any} value The value of the filter prop
  * @returns {Array}
  */
-const sGetDashboardsByProp = (state, propName, value) =>
-    Object.values(orObject(sGetById(state))).filter(
+const sGetDashboardsFilteredByPropValue = (state, propName, value) =>
```

Comment:
Should we really keep this function? It is not exported/used anywhere else in the app. Kind of confusing at first bc of the selector-like name and the docblock (which I know you didn't write...)

Instead the functions below that use it could just write the compact code like:

Object.values(sGetAllDashboards(state)).filter(d => d.starred === true);

Reply:
The idea is just that there might be a bunch of those coming later? So to avoid repeating "Object.values(sGetAllDashboards(state)).filter ...etc)" there is a generic function doing this instead.

Reply:
Not important, we could do that later if needed instead.